library-integration-slack/build.gradle

```diff
@@ -53,6 +53,8 @@ dependencies {
             "org.junit.jupiter:junit-jupiter-params:${junitJupiterVersion}")
     testCompile("com.nhaarman.mockitokotlin2:mockito-kotlin:${mockitoKotlinVersion}")
     testCompile("info.novatec.testit:testutils-logrecorder-logback:${logRecorderVersion}")
+    testCompile("com.github.tomakehurst:wiremock-standalone:${wiremockVersion}")
```

Comment:
Since you are using `spring-cloud-contract-wiremock` the `wiremock-standalone` dependency is already inherited.

Reply:
It is updated now 👍 I still might migrate to wiremock-standalone pure usage. I saw your WireMockExtension in the library-enrichment, still need to have a more detailed overview on the pros and cons of using the Autoconfiguration from Spring Cloud Contract vs custom one extension.

Reply:
then you might also be interested in this:
https://github.com/nt-ca-aqe/ws-cloud-native-testing/blob/master/spring-cloud-feign/service-foo/src/test/kotlin/foo/gateways/bar/WireMockBeanIntegrationTest.kt

Reply:
in that example project I demonstrated 3 approaches:
- using a JUnit 5 extension
- using Spring Cloud Contract WireMock
- adding WireMock as a bean to the test context (my favourite at the moment)